File: app/src/main/java/com/satugenggam/aplikasi1/data/model/RegisterResponse.java
package com.satugenggam.aplikasi1.data.model;

import com.google.gson.annotations.SerializedName;

public class RegisterResponse{

	@SerializedName("message")
	private String message;

	@SerializedName("user")
	private User user;

	public String getMessage(){
		return message;
	}

	public User getUser(){
		return user;
	}
}

File: app/src/main/java/com/satugenggam/aplikasi1/data/model/RegisterResponse.kt
package com.satugenggam.aplikasi1.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class RegisterResponse(

	@field:SerializedName("message")
	val message: String? = null,

	@field:SerializedName("user")
	val user: UserRegisterResponse? = null
): Parcelable
